updatecli/updatecli.d/terraform-providers/vpc.yaml

```diff
@@ -0,0 +1,50 @@
+name: Bump version of the Terraform "vpc" module in terraform-aws-sponsorship/vpc.tf
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  getLatestVersion:
+    kind: githubrelease
+    name: Retrieve the latest version
+    spec:
+      owner: "terraform-aws-modules"
+      repository: "terraform-aws-vpc"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionfilter:
+        kind: semver
+    transformers:
+      - trimprefix: v
+
+targets:
+  upgradeModuleVersion:
+    name: Update the Terraform "vpc" module version of terraform-aws-sponsored in vpc.tf
+    kind: file
+    sourceid: getLatestVersion
+    spec:
+      file: vpc.tf
+      matchpattern: >-
+        version(.*)=(.*)
+      replacepattern: >-
+        version = "{{ source `getLatestVersion` }}"
+    scmid: default
```

Comment:
```suggestion
    kind: hcl
    sourceid: getLatestVersion
    spec:
      file: vpc.tf
      path: module.vpc.version
    scmid: default
```

=> let's use native HCL resource instead of doing complex regex ;)

As the adage says, if you solve a problem with a regex, then you have 2 problems